xml/System.Threading/LazyInitializer.xml

```diff
@@ -114,7 +114,7 @@ ExpensiveData dataToUse = LazyInitializer.EnsureInitialized(ref _data, ref _data
       </Parameters>
       <Docs>
         <typeparam name="T">The type of the reference to be initialized.</typeparam>
-        <param name="target">A reference to initialize if it has not already been initialized.</param>
+        <param name="target">A reference to initialize if it has not already been initialized. If it is <see langword="null" />, it is considered not initialized.</param>
```

Comment:
Slightly more correct would be to say that it's considered not initialized if and only if it's null.  As written it speaks to the "if it's null" aspect, but not the "only if it's null" aspect.

Reply:
Added "; otherwise, it's considered not initialized"